### PR TITLE
Database index

### PR DESCRIPTION
**Pull request by armandodecanha**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -2,4 +2,9 @@
 
 * [Introduction](README.md)
 * [Front_End](./front-docs/LANDING_PAGE.md)
-* [Main_Page](./front-docs/MAIN_PAGE.md) 
+* [Main_Page](./front-docs/MAIN_PAGE.md)
```
Comment (delirial) [inline SUMMARY.md:5]: I think that the underscores are a little ugly xD

File: back-docs/DATABASE_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Getting Started 
+
+Setup and installation of the database and other libraries.
+
+1. ```git clone https://github.com/fullstacktf/myment-back```
+2. ```cd myment-back```
+3. ```npm run install```
```
Comment (delirial) [inline back-docs/DATABASE_SETUP.md:7]: Not exist an npm script for this purpose yet, but it can be good!

**Review by delirial — APPROVED** [review]
Very good!!! 🚀  some important information ! Thanks for that ! 🥇

File: SUMMARY.md (hunk 1 above)
Comment (armandodecanha) [reply SUMMARY.md:5]: Those underscores are NOT mine. They where there when I clone the repo.

**Pull request merged**